### ISSUE TITLE
2021 LTS + WebGL issues with wr.AbortController?

### ISSUE THREAD

**Issue by dbirman** [Benedicht/BestHTTP-Issues#119]
I'm getting the following exception building for WebGL on 2021.3.8f1 with the BestHTTP/2 package installed (although I'm not actually using it for anything). Removing the package causes the error to go away. Any thoughts on where this is coming from would be appreciated, I'd really like to implement some socket.io code in my builds!

```
exception thrown: TypeError: wr.abortControllers is undefined,_JS_WebRequest_Create@http://localhost:52026/Build/IBLMini-webgl.framework.js.gz:3:62385
```

**Comment by Benedicht** [comment]
What version of the plugin are you using? (You can check/print the `BestHTTP.HTTPManager.UserAgent` field.)
This issue might be fixed in [v2.6.2](https://benedicht.github.io/BestHTTP-Documentation/pages/best_http2/changelog.html#262-2022-04-18).

**Comment by dbirman** [comment]
Ah, didn't even realize I was out of date. Updating to 2.7 appears to have fixed this, thanks for the help!
